routers/routes/routes.go

```diff
@@ -523,6 +523,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+
+		m.Group("/branches", func() {
+			m.Get("/_new/*", context.RepoRef(), repo.CreateBranch)
```

Comment:
Just to clarify, I'm fairly certain that no one would create a branch called "new" anyhow 😆